src/Mod/TechDraw/App/Geometry.cpp

```diff
@@ -1330,12 +1328,7 @@ void Vertex::Save(Base::Writer &writer) const
     writer.Stream() << writer.ind() << "<Cosmetic value=\"" <<  c2 << "\"/>" << endl;
     writer.Stream() << writer.ind() << "<CosmeticLink value=\"" <<  cosmeticLink << "\"/>" << endl;
     writer.Stream() << writer.ind() << "<CosmeticTag value=\"" <<  cosmeticTag << "\"/>" << endl;
-
-    //do we need to save this?  always recreated by program.
-//    const char r = reference?'1':'0';
-//    writer.Stream() << writer.ind() << "<Reference value=\"" <<  r << "\"/>" << endl;
-
-    Tag::Save(writer);
+    writer.Stream() << writer.ind() << "<VertexTag value=\"" <<  getTagAsString() << "\"/>" << endl;
```

Comment:
We need to keep sacing it as `<Tag />` so we can deprecate `<VertexTag />` sometime in the future and simplify logic. The solution should be `Tag::Restore` both accepting `<Tag />` and `<VertexTag />`.

Reply:
If you have a Tag::Save here and another one in CosmeticVertex, you'll get 2 Tag entries in the xml.   When the time comes to remove VertexTag, we'll delete lines 1331 and 1355.